fixtures/pages-simple-assets/tests/index.test.ts

```diff
@@ -1,21 +1,18 @@
 import { resolve } from "node:path";
 import { fetch } from "undici";
-import { describe, it, beforeAll, afterAll } from "vitest";
+import { describe, it, afterAll } from "vitest";
 import { runWranglerPagesDev } from "../../shared/src/run-wrangler-long-lived";
 
-describe.concurrent("Pages Functions", () => {
-	let ip, port, stop;
-
-	beforeAll(async () => {
-		({ ip, port, stop } = await runWranglerPagesDev(
-			resolve(__dirname, ".."),
-			"public",
-			[]
-		));
+describe("Pages Functions", async () => {
+	const { ip, port, stop } = await runWranglerPagesDev(
+		resolve(__dirname, ".."),
+		"public",
+		[]
+	);
+	afterAll(async () => {
+		await stop();
```

Comment:
The `beforeAll` hook was hanging, probably something to do with the way test hooks handle async/await, moving the awaited "server" function outside the hook and awaiting solved the hanging problem. 

Likely a brittle test to begin with that surfaced in this environment.

Reply:
I wonder if this is due to the test itself being run in a different scope? Perhaps inside a different thread/process, which means that running wrangler doesn't work?
I note that the `stop is not a function` error is a red-herring - it is not a function because the call to `runWranglerPagesDev` never resolves so `stop` is not assigned.

Reply:
The `pages-functions-app:test` job was timing out, so I changed the `runWranglerPagesDev()` function to inherit the stdout and stderr, so we could see what is going on. It turns out that this run of wrangler was hanging because:

```
pages-functions-app:test:   mounted-plugin/_middleware.ts:1:26: ERROR: Could not resolve "../../../pages-plugin-example"
pages-functions-app:test:   mounted-with-param/[p]/plugin/_middleware.ts:1:26: ERROR: Could not resolve "../../../../../pages-plugin-example"
```

so it was never getting to the IPC message send, which is why it was hanging.

Reply:
So the solution is to make `pages-plugin-example` a dependency (in `package.json`) and then change the import to from

```ts
import examplePlugin from "../../../pages-plugin-example";
```

to 

```ts
import examplePlugin from "pages-plugin-example";
```